### PR TITLE
Dashboard widget: change messaging

### PR DESCRIPTION
**Pull request by richardmuscat**
Making messaging simpler and consistent as per discussion on P2

This change includes:
- A call to action that says “Connect Jetpack” to match with what we’re
asking the user to do
- A heading saying “Please Connect Jetpack”
- A simpler line of descriptive text highlighting the key features

### CODE REVIEW

File: class.jetpack.php [hunk 1]
```diff
@@ -5956,14 +5956,16 @@ public static function dns_prefetch( $new_urls = null ) {
 	public function wp_dashboard_setup() {
 		if ( self::is_active() ) {
 			add_action( 'jetpack_dashboard_widget', array( __CLASS__, 'dashboard_widget_footer' ), 999 );
+			$widget_title = __( "Jetpack", 'jetpack' );
```
Comment (dereksmart) [inline class.jetpack.php]: translation function uses both double quotes and single quotes.  Let's just go single quotes.

File: class.jetpack.php [hunk 2]
```diff
@@ -5956,14 +5956,16 @@ public static function dns_prefetch( $new_urls = null ) {
...
 		} elseif ( ! self::is_development_mode() && current_user_can( 'jetpack_connect' ) ) {
 			add_action( 'jetpack_dashboard_widget', array( $this, 'dashboard_widget_connect_to_wpcom' ) );
+			$widget_title = __( "Please Connect Jetpack", 'jetpack' );;
```
Comment (dereksmart) [inline class.jetpack.php]: There's an extra semicolon

File: class.jetpack.php [hunk 3]
```diff
@@ -6059,18 +6061,13 @@ public function dashboard_widget_connect_to_wpcom() {
   				<path d="M86.4 0C38.7 0 0 38.7 0 86.4c0 47.7 38.7 86.4 86.4 86.4s86.4-38.7 86.4-86.4C172.9 38.7 134.2 0 86.4 0zM83.1 106.6l-27.1-6.9C49 98 45.7 90.1 49.3 84l33.8-58.5V106.6zM124.9 88.9l-33.8 58.5V66.3l27.1 6.9C125.1 74.9 128.4 82.8 124.9 88.9z"/>
 			</svg>
 			</div>
-			<h3><?php esc_html_e( 'Boost traffic, enhance security, and improve performance.', 'jetpack' ); ?></h3>
-			<p><?php esc_html_e( 'Jetpack connects your site to WordPress.com to give you traffic and customization tools, enhanced security, speed boosts, and more.', 'jetpack' ); ?></p>
+			<h3><?php esc_html_e( 'Please Connect Jetpack', 'jetpack' ); ?></h3>
+			<p><?php echo wp_kses( __( 'Connecting Jetpack will show you <strong>stats</strong> about your traffic, <strong>protect</strong> you from brute force attacks, <strong>speed up</strong> your images and photos, and enable other <strong>traffic and security</strong> features.' ), 'jetpack' ) ?></p>
```
Comment (dereksmart) [inline class.jetpack.php]: let's use sprintf() or printf() to insert the html tags

File: class.jetpack.php (hunk 2 above)
Comment (richardmuscat) [inline class.jetpack.php]: Fixed in https://github.com/Automattic/jetpack/commit/56e77efc2c40e3f5436d70af80cbb2f276f581c1

File: class.jetpack.php (hunk 1 above)
Comment (richardmuscat) [inline class.jetpack.php]: Fixed in https://github.com/Automattic/jetpack/commit/dd62a9cf29c74f7a6d33b980b89cef22d0185f8d

File: class.jetpack.php (hunk 3 above)
Comment (richardmuscat) [inline class.jetpack.php]: @dereksmart not sure about the whys and hows. @zinigor suggested wp_kses as that seems to be the recommended way by i18n crew

Comment (dereksmart) [inline class.jetpack.php]: ok wfm, I only said something because i18n wanted me to sprintf() the HTML for 3.6.

**Comment by zinigor** [comment]
Everything looks good!

**Comment by richardmuscat** [comment]
Woot

**Pull request merged**